_includes/payment-presentations.md

```diff
@@ -0,0 +1,185 @@
+---
+section: Checkout v3
+title:
+description: |
+  **An overview of all the payment instruments Checkout v3 has to offer.**
+checkout_v3: true
+menu_order: 400
+---
+
+## Apple Pay
+
+Apple Pay provides an easy and secure way to make payments. By using Face ID,
+Touch ID, or double-clicking their Apple Watch, payers can quickly and securely
+check out. Payers love the simplicity, and you’ll love the increased conversion
+rates and new user adoption that comes with it. There are some additional steps
+steps which must be completed befire we can activate Apple Pay for you.
+
+### Domain Verification
+
+**Redirect** integrations does not require a domain verification, as it will
+be hosted on Swedbank Pay's domain.
+
+For **Seamless View** integrations, Apple needs to verify your domain as a
+part of Swedbank Pay's set up process. To do this, you need to host a
+verification file on the following web path:
+
+https://[DOMAIN_NAME]/.well-known/apple-developer-merchantid-domain-association
```

Comment:
Since this URL is supposed to be read as code, we should declare it as code too.

````suggestion
```
https://[DOMAIN_NAME]/.well-known/apple-developer-merchantid-domain-association
```
````